modules/home-manager/home.nix

```diff
@@ -0,0 +1,30 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+let
+  inherit (lib) ctp mkIf;
+  cfg = config.home.catppuccin;
+in
+{
+  options.home.catppuccin.cursor =
+    ctp.mkCatppuccinOpt ''
+      global cursors. Note that these options are overwritten by the options in
+      `gtk.catppuccin.cursor`
+    ''
+    // {
+      accent = ctp.mkCursorAccentOpt "global cursors";
+    };
+
+  config.home.pointerCursor =
+    let
+      flavorUpper = ctp.mkUpper cfg.cursor.flavor;
+      accentUpper = ctp.mkUpper cfg.cursor.accent;
+    in
+    mkIf cfg.cursor.enable {
+      name = "Catppuccin-${flavorUpper}-${accentUpper}-Cursors";
```

Comment:
I believe the name was changed to all lowercase as of catppuccin cursors v0.2.1. Might be a good idea to set the name based on the version of the package.

Reply:
we need to support 24.05 which is still on v0.2.0

Reply:
um opsies... should of thought about that before https://github.com/catppuccin/nix/pull/212. sigh...

Reply:
https://github.com/NixOS/nixpkgs/pull/321759